src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstLeg;
```

Comment:
Consider renaming 'firstLeg' to 'width' for better readability and understanding 📏